upgrade/rendering-changes/4-3-0.md

```diff
@@ -21,7 +21,51 @@ position: 1
 
 The default icon type in the Telerik UI for Blazor suite changes from `FontIcon` to `SvgIcon`. This marks the next milestone in a series of improvements related to the [Content Security Policy (CSP)]({%slug troubleshooting-csp%}).
 
->note As part of the R3 2023 release, the font icons will be detached from the Telerik UI for Blazor themes. To continue using the font icons in your application, either add a separate CDN reference or download and reference the stylesheet included in the [Telerik UI for Blazor package]({%slug installation/msi%}##how-to-download-the-automated-installer).
+### All Telerik UI for Blazor components will now use SVG icons by default
+
+When upgrading your application to 4.3.0 you might experience two possible scenarios related to the new default icon type:
+
+* [Custom CSS that targets icons may stop working](#changes-in-the-rendering)
+* [Failing HTML tests](#failing-html-tests)
+
+#### Changes In The Rendering
+
+The built-in icons in the Telerik UI for Blazor components will now render with different CSS classes which will break any CSS selectors that are customizing the icons:
```

Comment:
```suggestion
The built-in icons in Telerik UI for Blazor will now render with different HTML markup and CSS classes. This will break any CSS selectors that are targeting the icons:
```